Misc/NEWS.d/next/Core and Builtins/2020-01-13-14-45-22.bpo-39048.iPsj81.rst

```diff
@@ -0,0 +1,2 @@
+Change the lookup order of the :meth:`__aenter__` and :meth:`__aexit__` methods
```

Comment:
Change overall looks good to me, but the NEWS entry would benefit from a tweak to better convey *what* changed.

Reply:
Something like:

> To improve the displayed error message when incorrect types are passed to `async with` statements, the :meth:`__aenter__` special method is now looked up before the :meth:`__aexit__` method when entering an async context manager.

Reply:
@ncoghlan Done!